_includes/components/header--basic.html

```diff
@@ -0,0 +1,72 @@
+{% if header %}
+<header class="usa-header usa-header-{{ header.type | default: 'extended' }}" role="banner">
```

Comment:
Last little nitpick - since this is now in it's own `--basic.html` file I think this could just be `usa-header-basic` instead of `{{ header.type | default: 'extended' }}`.

Reply:
Thanks @hursey013 pushing an update to this branch now! 🎉